AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/OutboxMutationEvent.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import AWSPluginsCore
+
+/// Used as HubPayload for `OutboxMutationEnqueued` and `OutboxMutationProcessed`
+public struct OutboxMutationEvent {
+    public var modelName: String
+    public var element: OutboxMutationEventElement
+
+    public static func fromModelWithMetadata(modelName: String,
+                                             model: Model,
+                                             mutationSync: MutationSync<AnyModel>) -> OutboxMutationEvent {
+        let element = OutboxMutationEventElement(model: model,
+                                                 version: mutationSync.syncMetadata.version,
+                                                 lastChangedAt: mutationSync.syncMetadata.lastChangedAt,
+                                                 deleted: mutationSync.syncMetadata.deleted)
+        return fromModel(modelName: modelName, element: element)
+    }
+
+    public static func fromModelWithoutMetadata(modelName: String,
+                                                model: Model) -> OutboxMutationEvent {
+        let element = OutboxMutationEventElement(model: model)
+        return fromModel(modelName: modelName, element: element)
+    }
+
+    public static func fromModel(modelName: String, element: OutboxMutationEventElement) -> OutboxMutationEvent {
+        return OutboxMutationEvent(modelName: modelName, element: element)
+    }
```

Comment:
this looks like a constructor, any reason for static method?
```
pubic init(modelName: String, element: OutboxMutationEventElement) {
 self.modelName = modelName
self.element = element
}

Reply:
Hmmm. Totally right 😂 I will remove this method and add an `init()`